app/assets/javascripts/components/user_components/user_overview_info.js.jsx

```diff
@@ -0,0 +1,26 @@
+function Info(props) {
+        return (
+            <div className={"row col-sm-10"}>
```

Comment:
don't use `row` with `col` for same element
[see doc for more details](https://v4-alpha.getbootstrap.com/layout/grid/#equal-width)